dotcom-rendering/docs/patterns/enhance-capi.md

```diff
@@ -2,23 +2,23 @@
 The enhance CAPI pattern is one where the CAPI object is transformed into a different, more useable, format. In particular, it relates to the `blocks` array. Each enhancement function, takes the `CAPIType` and returns the `CAPIType`.
 
 ## Lexicon
-An 'enhancement' could also be termed a 'transformation' and in other places similar functions are refrered to as 'cleaners'.
+An 'enhancement' could also be termed a 'transformation' and in other places similar functions are refereed to as 'cleaners'.
 
 ## Why?
 Ideally, these transformations would not be required. A better solution would be to construct the array at source, in Composer or CAPI, with the ideal structure. But for - probably very good - reasons the decision was taken to use [cleaners in frontend](https://github.com/guardian/frontend/blob/aa0013a6f9c247be36d29b9716e0ccc80cc8b218/common/app/views/support/HtmlCleaner.scala) to solve design problems. So now, in order to support the same designs, we need to replicate these cleaners in DCR. The enhancements are applied in the [server/index.ts](/dotcom-rendering/src/web/server/index.ts).
 
 ## Examples
 
 ### DropCaps
-Certain article types, such as Features, have the first letter of the first paragraph marked as a DropCap. However, there was a requirement to sometimes also have other paragraphs be given a DropCap. But composer does not offer a way to mark a paragraph as, say, dropcap: true, so instead a convention was invented where if the preceeding element was a h2 tag containing '* * *' then that was the trigger to give the following paragraph drop cap styling.
+Certain article types, such as Features, have the first letter of the first paragraph marked as a DropCap. However, there was a requirement to sometimes also have other paragraphs be given a DropCap. But composer does not offer a way to mark a paragraph as, say, `dropcap: true`, so instead a convention was invented where if the preceding element was a h2 tag containing '* * *' then that was the trigger to give the following paragraph drop cap styling.
 
 In DCR we replicate this using [enhance-dividers.ts](/dotcom-rendering/src/model/enhance-dividers.ts)
 
 ### Images
 There are some conventions that can result in images appearing differently, we need a cleaner to support these while we wait for support for them to be added natively to Composer.
 
 Multi images. Consecutive sequences of two halfWidth images will be merged into a MultiImageBlockElement and shown side by side
-Captions. A ul/li tag directly after an image will replace the preoceeding image's caption
+Captions. A ul/li tag directly after an image will replace the proceeding image's caption
```

Comment:
I don't fully understand what this means.

-`img` tag A
-`ul`/`li` tag
-`img` tag B

Which image's caption does the `ul`/`li` tag replace?